Readability.js

```diff
@@ -844,18 +844,50 @@ Readability.prototype = {
 
         this._initializeNode(topCandidate);
       } else if (topCandidate) {
+        parentOfTopCandidate = topCandidate.parentNode;
+
+        // Find a better top candidate node if it contains (at least three) nodes which belong to `topCandidates` array
+        // and whose scores are quite closed with current `topCandidate` node.
+        var alternativeCandidateAncestors = [];
+        for (var i = 1; i < topCandidates.length; i++) {
```

Comment:
I would say this for loop is clear enough. I think we don't need to add new method `lookForSharedAncestorAsTopCandidate` for clarity.